.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -30,6 +30,12 @@ If you are running into one of these scenarios, we recommend opening an issue in
 
 <!--- Please run `terraform -v` to show the Terraform core version and provider version(s). If you are not running the latest version of Terraform or the provider, please upgrade because your issue may have already been fixed. [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions). --->
 
+### Has this been tested with the latest version of Terraform?
```

Comment:
If we are going to ask this question, I would suggest a few things:

- Removing the verbiage above about "If you are not running the latest...", or more preferably, moving that comment below this question
- Also asking about the latest version of the Terraform AWS Provider, which is versioned differently. 👍
- The next question seems like it could also be added here for consolidation and to dig deeper into potential issues people may have with upgrading in general.

```suggestion
### Has this been tested with the latest version of Terraform CLI and Terraform AWS Provider? If you cannot upgrade, please let us know why.
```

Reply:
That makes sense. I was trying to keep it short, but it flows better this way.